### ISSUE TITLE
static just like const can only be literals

### ISSUE THREAD

**Issue by mardaravicius** [Divy1211/xs-check#5]
```c
static int s = xsGetRandomNumber();
```

would produce an error
```
Error 0334 `xsGetRandomNumber` is not a valid variable for an integer constant.
```
but the checker says it's fine

 of course, unlike const, they can be changed to another value that is not a literal
 ```c
 static int s = 0;
 s = xsGetRandomNumber(); /* this is fine */
 ```

**Comment by Divy1211** [comment]
That makes sense, a static variable has program lifetime, it must be defined by a constexpr. I can add this check, thanks for the report

**Comment by Divy1211** [comment]
Fixed in v0.1.3
